relational_operators/HashJoinOperator.cpp

```diff
@@ -63,6 +65,9 @@ namespace quickstep {
 
 namespace {
 
+typedef std::vector<std::pair<tuple_id, tuple_id>> VectorOfPairs;
+typedef std::pair<std::vector<tuple_id>, std::vector<tuple_id>> PairOfVectors;
```

Comment:
Suggest to rename to `VectorOfTupleIdPair` and `PairOfTupleIdVector`, respectively.